test/upgradeability/OwnedUpgradeabilityProxy.js

```diff
@@ -1,38 +1,25 @@
 'use strict'
 
 const assertRevert = require('../helpers/assertRevert')
-const Registry = artifacts.require('Registry')
-const Factory = artifacts.require('Factory')
 const OwnedUpgradeabilityProxy = artifacts.require('OwnedUpgradeabilityProxy')
+const UpgradeabilityProxyFactory = artifacts.require('UpgradeabilityProxyFactory')
 
-contract('OwnedUpgradeabilityProxy', ([owner, anotherAccount, implementation_v0, implementation_v1]) => {
+contract('OwnedUpgradeabilityProxy', ([_, owner, anotherAccount, implementation_v0, implementation_v1]) => {
```

Comment:
The reason we usually add `_` as the first variable is that Truffle will use this address by default, and the times that we care about the sender being a specific address we want to force ourselves to be explicit about it.

In this case, for example, all transactions without an explicit sender will be sent by the owner, and this might have undesirable results.

Reply:
Oops, I misread the diff and thought you were removing the `_`! My bad.